flink-table/flink-sql-client/src/test/resources/sql/set.q

```diff
@@ -118,47 +118,61 @@ SELECT hive_add_one(1);
 Received a total of 1 row
 !ok
 
-REMOVE JAR '$VAR_UDF_JAR_PATH';
-[INFO] The specified jar is removed from session classloader.
-!info
+# TODO: support this in the FLINK-30692
+#REMOVE JAR '$VAR_UDF_JAR_PATH';
+#[INFO] The specified jar is removed from session classloader.
+#!info
 
-SHOW JARS;
-Empty set
-!ok
+#SHOW JARS;
+#Empty set
+#!ok
+
+reset table.resources.download-dir;
+[INFO] Execute statement succeed.
+!info
 
 # list the configured configuration
 set;
-'execution.attached' = 'true'
-'execution.savepoint-restore-mode' = 'NO_CLAIM'
-'execution.savepoint.ignore-unclaimed-state' = 'false'
-'execution.shutdown-on-attached-exit' = 'false'
-'execution.target' = 'remote'
-'jobmanager.rpc.address' = '$VAR_JOBMANAGER_RPC_ADDRESS'
-'k1' = 'v1'
-'pipeline.classpaths' = ''
-'pipeline.jars' = ''
-'rest.port' = '$VAR_REST_PORT'
-'sql-client.execution.result-mode' = 'tableau'
-'table.exec.legacy-cast-behaviour' = 'DISABLED'
-'table.sql-dialect' = 'hive'
++--------------------------------------------+-----------+
+|                                        key |     value |
++--------------------------------------------+-----------+
+|                         execution.attached |      true |
+|           execution.savepoint-restore-mode |  NO_CLAIM |
+| execution.savepoint.ignore-unclaimed-state |     false |
+|        execution.shutdown-on-attached-exit |     false |
+|                           execution.target |    remote |
+|                     jobmanager.rpc.address | $VAR_JOBMANAGER_RPC_ADDRESS |
+|                                         k1 |        v1 |
+|                        pipeline.classpaths |           |
+|                              pipeline.jars |           |
+|                                  rest.port |     $VAR_REST_PORT |
+|           sql-client.execution.result-mode |   tableau |
+|           table.exec.legacy-cast-behaviour |  DISABLED |
+|                          table.sql-dialect |      hive |
++--------------------------------------------+-----------+
+13 rows in set
```

Comment:
Thanks for pointing it out! I find I use the wrong API in the ExecutorImpl. I should use `ScheduledExecutorService#scheduleAtFixedRate` rather than `ScheduledExecutorService#schedule`. The latter API only triggers the session once.